R/zchunk_LB126.Gas_ElecTD.R

```diff
@@ -308,9 +308,9 @@ module_gcam.usa_LB126.Gas_ElecTD <- function(command, ...) {
       add_legacy_name("L126.out_EJ_state_td_elec") %>%
       add_precursors("L122.in_EJ_state_refining_F",
                      "L123.out_EJ_state_elec_F",
-                     "temp-data-inject/L132.in_EJ_state_indchp_F",
-                     "temp-data-inject/L132.in_EJ_state_indfeed_F",
-                     "temp-data-inject/L132.in_EJ_state_indnochp_F",
```

Comment:
Thanks. Did you delete these files out of `inst/extdata/temp-data-inject/`?

Reply:
Done now. I'd also left parts of the temp-inject code in the chunk, which was causing most of the oldnew fails.